web-ng/src/app/components/project-header/project-header.component.html

```diff
@@ -13,16 +13,23 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 -->
-
+<img class="logo" src="/assets/img/logo.png" />
+<span class="ground"> Ground </span>
+<span> | </span>
```

Comment:
Can we style this with CSS instead to make it look more polished and closer to the mocks?